app/models/base_facility.rb

```diff
@@ -81,9 +81,10 @@ def find_facility_by_id(id)
     # FIXME: using common collection here to have access to #metadata for serializer output but long term we should
     # refactor and mix that in -- https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9376
     def query(params)
+      return common_collection([]) unless params[:bbox]
```

Comment:
I'm afraid this common collection won't have the correct type. Might run into javascript parsing issues without or schema failures. Do you have a spec for this scenario?

Reply:
This was added to replicate original behavior and make the swagger specs pass. In practice the bbox param is required. I'm up for changing this behavior in a future PR but felt like we should make the switchover with matching functionality.

Reply:
probably ok if type is set in the serializer.